lib/resque/scheduler.rb

```diff
@@ -377,7 +377,6 @@ def handle_signals_with_operation
 
       def stop_rufus_scheduler
         rufus_scheduler.shutdown(:wait)
-        rufus_scheduler.join
```

Comment:
Looking at the CI tests, looks like the only version failing is `3.6`. 

I'd suggest something like:

```ruby
rufus_scheduler.join if Gem.loaded_specs["rufus_scheduler"].version.to_s.starts_with?("3.6")
```

I'd guess this change would potentially fix the test.

Reply:
@fagiani ugh, looks like a flake, rebuilt and it passed.  Ran it locally without failure.  I don't believe that failure would even be related to this change though.

Reply:
great @carsonreinke ! 

@iloveitaly when is the next release including this PR planned?

Keep Rocking!